docs/configuration-reference/components/cluster-autoscaler.md

```diff
@@ -95,10 +95,10 @@ lokoctl component apply cluster-autoscaler
 ```
 By default, the cluster Autoscaler pods run in the `kube-system` namespace
 
-## Destroying
+## Deleting
 
 To destroy the component:
 
 ```bash
-lokoctl component render-manifest cluster-autoscaler | kubectl delete -f -
+lokoctl component delete cluster-autoscaler
```

Comment:
This does not have `--delete-namespace` flag because it is deployed in `kube-system` namespace.

Reply:
Although a user cannot delete that namespace, it is not a problem.

Reply:
What happens if the user specifies `--delete-namespace` for a component which doesn't support NS deletion?

Reply:
So as to make this feature idempotent we decided we don't error out on deleting a namespace, because user can try again to delete it. So it will fail but user won't see anything.

By not specifying that flag I want to avoid namespace deletion trigger.